python-lib/plugin_io_utils.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+"""Module with read/write utility functions based on the Dataiku API"""
+
+from typing import List, AnyStr, Union
+
+import pandas as pd
+import numpy as np
+
+import dataiku
+import logging
+
+
+def count_records(dataset: dataiku.Dataset) -> int:
```

Comment:
Can you use the latest version: https://github.com/dataiku/dss-plugin-nlp-preparation/blob/master/python-lib/dku_io_utils.py

Reply:
not used so removed